docs/source/guide/tasks.md

```diff
@@ -204,7 +216,11 @@ Use that script to do the following:
    The script collects the links to the files provided by that HTTP server and saves them to a `files.txt` file with one URL per line. 
 3. Import the file with URLs into Label Studio using the Label Studio UI. 
 
-To import tasks from a file directory on a remote server, make sure the remote server is accessible to the server running Label Studio and set up the directory as a [local storage](storage.html#Local-storage) connection. 
+
+### Add the file directory as source storage in the Label Studio UI
+To import tasks from a file directory on a remote or local server, make sure the remote server is accessible to the server running Label Studio and set up the directory as a [local storage](storage.html#Local-storage) connection. See [local storage](storage.html#Local-storage).
```

Comment:
I don't think this could be a real use case with connecting file storage from a remote server. Better to avoid that

Reply:
I'll remove it! thanks